Sources/Subs-Sound.php

```diff
@@ -65,32 +65,32 @@ function createWaveFile($word)
 			return false;
 
 		$sound_letter = substr($sound_letter, strpos($sound_letter, 'data') + 8);
-		switch ($word{$i} === 's' ? 0 : mt_rand(0, 2))
+		switch ($word{$i} === 's' ? 0 : random_int(0, 2))
 		{
 			case 0 : for ($j = 0, $n = strlen($sound_letter); $j < $n; $j++)
-					for ($k = 0, $m = round(mt_rand(15, 25) / 10); $k < $m; $k++)
-						$sound_word .= $word{$i} === 's' ? $sound_letter{$j} : chr(mt_rand(max(ord($sound_letter{$j}) - 1, 0x00), min(ord($sound_letter{$j}) + 1, 0xFF)));
+					for ($k = 0, $m = round(random_int(15, 25) / 10); $k < $m; $k++)
+						$sound_word .= $word{$i} === 's' ? $sound_letter{$j} : chr(random_int(max(ord($sound_letter{$j}) - 1, 0x00), min(ord($sound_letter{$j}) + 1, 0xFF)));
 			break;
 
 			case 1:
 				for ($j = 0, $n = strlen($sound_letter) - 1; $j < $n; $j += 2)
-					$sound_word .= (mt_rand(0, 3) == 0 ? '' : $sound_letter{$j}) . (mt_rand(0, 3) === 0 ? $sound_letter{$j + 1} : $sound_letter{$j}) . (mt_rand(0, 3) === 0 ? $sound_letter{$j} : $sound_letter{$j + 1}) . $sound_letter{$j + 1} . (mt_rand(0, 3) == 0 ? $sound_letter{$j + 1} : '');
+					$sound_word .= (random_int(0, 3) == 0 ? '' : $sound_letter{$j}) . (random_int(0, 3) === 0 ? $sound_letter{$j + 1} : $sound_letter{$j}) . (random_int(0, 3) === 0 ? $sound_letter{$j} : $sound_letter{$j + 1}) . $sound_letter{$j + 1} . (random_int(0, 3) == 0 ? $sound_letter{$j + 1} : '');
```

Comment:
This change is not needed.This change is not needed.This change is not needed.This change is not needed.